client/pages/components/InputTextBox.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Tippy from '@tippy.js/react';
+import 'tippy.js/dist/tippy.css';
+import store from '../../store';
+
+export default function InputTextBox({ label, rows, placeholder, text}) {   
+  const [state, setState] = useState(text);
+
+  async function handleChange(e) {
+    const change = e.target.value;
+    await setState(change);
+    store.dispatch({
+      type: `set${label}`,
+      payload: change
+    });
+  }
+
```

Comment:
Interesting! It both manages its own state and updates the store. Is it more efficient to manage its state locally?
There should also be a way to subscribe to the Redux store's `set${label}`. No harm in doing it this way, but a "single source of truth" may be helpful in other cases?